scrapy/settings/default_settings.py

```diff
@@ -258,6 +258,7 @@
     'scrapy.spidermiddlewares.referer.RefererMiddleware': 700,
     'scrapy.spidermiddlewares.urllength.UrlLengthMiddleware': 800,
     'scrapy.spidermiddlewares.depth.DepthMiddleware': 900,
+    'scrapy.spidermiddlewares.retry.RetryMiddleware': 950,
```

Comment:
This can cause some confusion because now there are two `RetryMiddlewares`.
One is a spider middleware and the other one is a downloader middleware.